server.rb

```diff
@@ -1,8 +1,14 @@
 require 'erubis'
 require 'json'
 require 'sinatra'
+require_relative 'lib/badge'
+require_relative 'lib/member'

```

Comment:
Any reason to `require` those 2 here instead of in `leaderboard.rb` ? Just asking, I don' have much experience in Ruby.

Reply:
No reason, actually that's barely the same, it depends on where you need to use that classes. In this case Badge and Member are strictly related to leaderboard so maybe it's more correct to move them.